docs/decisions/0013-library-reference-content-block.rst

```diff
@@ -25,30 +25,30 @@ Terminology Used
 
 Decisions
 =========
-We want to be able to reference single and multiple specific blocks from a library (v2, as well as v1) into a course (that are stored in modulestore)
+The requirement is to reference single and multiple specific blocks from a library (v2, as well as v1) into a course (that are stored in modulestore)
 The workflow discussed for the interface for library content referencing would be:
 
 #. Author chooses a library.
-#. Author chooses a pool of blocks.
+#. Author can choose to reference one or multiple blocks.
 #. If multiple blocks of the same type are selected, ask for randomization
 #. If no randomization, then ask for ordering
 
 Current Architecture/Implementation
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 Currently, the courses are stored in modulestore and the libraries can either be stored in modulestore or blockstore.
-Since the course lives in the modulestore and some of the libraries in blockstore, we use the [library_sourced_block](https://github.com/openedx/edx-platform/blob/master/common/lib/xmodule/xmodule/library_sourced_block.py) to make a copy of that blockstore based library block and store it in modulestore itself as the child.
+Since the course lives in the modulestore and some of the libraries in blockstore, the [library_sourced_block](https://github.com/openedx/edx-platform/blob/master/common/lib/xmodule/xmodule/library_sourced_block.py) is used to make a copy of that blockstore based library block and store it in modulestore itself as the child.
 
 
 Proposed Implementation
 ^^^^^^^^^^^^^^
 The library referencing flow would be implemented within the Studio and/or LMS processes of a single Open edX instance.
 The library content would be stored and rendered outside the modulestore.
 
-We would achieve this by introducing a unit_compositor subsystem. Like learning_sequences, the subsystem would be populated by CMS upon course publish. It would store a read-optimized form of:
+This is achieved by a unit_compositor subsystem. Like learning_sequences, the subsystem would be populated by CMS upon course publish. It would store a read-optimized form of:
```

Comment:
Here "it" is the Modulestore right?

Reply:
I believe "it" here refers to the "unit_compositor" system, which is something new that doesn't exist yet.

Reply:
@farhaanbukhsh: @bradenmacdonald is right. it refers to `unit_compositor`